src/components/TextareaField/TextareaField.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { FieldWrapper, Textarea } from './styled';
+
+type CreateEventProps = {
+  name: string;
+  label: string;
+  value: string;
+  onChange: Function;
+};
+
+function TextareaField({ name, label, value, onChange }: CreateEventProps) {
```

Comment:
Consider renaming this to `TextAreaField`. I just feel that it looks more natural that way.